src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -510,12 +511,21 @@ impl Sandbox for VirtSandbox {
             warn!(sl!(), "sandbox is started");
             return Ok(());
         }
+        let selinux_label = load_oci_spec()
+        .ok()
+        .and_then(|spec| {
+            spec.process()
+                .as_ref()
+                .and_then(|process| process.selinux_label().clone())
+        })
+        .unwrap_or_default();
```

Comment:
Don't need `unwrap_or_default()`, so that the type of `selinux_label` will be `Option<String>. In this case we can give it to `perpare_vm()` directly. Right?